Command/RemoveThumbsCommand.php

```diff
@@ -140,53 +135,73 @@ public function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * @param string $providerName
+     *
      * @return MediaProviderInterface
      */
-    public function getProvider()
+    protected function getProvider($providerName)
```

Comment:
> Maybe, but this is not a parent class constraint and this is BC break.

@Soullivaneuh Do you mean I only have to change `getContext`, `getProvider` and `getFormat` methods back to public with the same arguments, or change the `execute` method too?